script/script_test.go

```diff
@@ -42,6 +43,15 @@ func (l *luaContext) ParentSpan() opentracing.Span          { return nil }
 func (l *luaContext) Split() (filters.FilterContext, error) { return nil, nil }
 func (l *luaContext) Loopback()                             {}
 
+func (l *luaContext) Logger() filters.FilterContextLogger {
+	traceId := tracing.GetTraceID(l.request)
+	if traceId == "" {
+		return log.StandardLogger()
+	} else {
+		return log.WithFields(log.Fields{"trace_id": traceId})
+	}
+}
```

Comment:
Lets get rid of luaContext altogether, see https://github.com/zalando/skipper/pull/2273

Reply:
Looks good to me. :-)